edward/models/random_variable.py

```diff
@@ -59,7 +59,8 @@ def __init__(self, *args, **kwargs):
     # need to temporarily pop value before __init__
     value = kwargs.pop('value', None)
     super(RandomVariable, self).__init__(*args, **kwargs)
-    self._kwargs['value'] = value  # reinsert (needed for copying)
+    if value is not None:
```

Comment:
Oh, and to clarify:

If one were to allow `kwargs['value'] = None`, then the subsequent copy tries to use `value=None` as an initializer, which would block the normal path (the one that calls `sample`).